chat-plugins/scavengers.js

```diff
@@ -300,4 +300,4 @@ exports.commands = {
 			'- /scavresetlb - Reset the official leaderboard (Require: # & ~)'
 		);
 	},
-};
```

Comment:
This is weird, because when I go in the file editor, it has an extra space at the end...